pkg/util/validation/limits.go

```diff
@@ -100,6 +100,9 @@ type Limits struct {
 	// Max allowed time window for out-of-order samples.
 	OutOfOrderTimeWindow model.Duration `yaml:"out_of_order_time_window" json:"out_of_order_time_window" category:"experimental"`
 
+	// Label to further separate specfic metrics by
+	SeparateMetricsLabel string `yaml:"separate_metrics_label" json:"separate_metrics_label"`
```

Comment:
Sometimes we say _split_, sometimes we say _separate_. We should use a common term everywhere.